sites/all/modules/publizon/includes/publizon.availability.inc

```diff
@@ -62,8 +73,8 @@ function publizon_availability_items(array $local_ids) {
 
   try {
     $client = PublizonLoanClient::getClient();
-    $loanStatus = $client->getProductLoanStatusList($retailer_id, array_keys($to_check), $card_number);
-    foreach ($loanStatus as $loanStatus) {
+    $loanStatusList = $client->getProductLoanStatusList($retailer_id, array_keys($to_check), $card_number);
```

Comment:
Should be `$loan_status_list` (cf. https://www.drupal.org/docs/develop/standards/coding-standards#functions-and-variables).

Reply:
Fixed